frontend/src/components/common/Resource/RestartButton.tsx

```diff
@@ -0,0 +1,126 @@
+import { Icon } from '@iconify/react';
+import {
+  Button,
+  Dialog,
+  DialogActions,
+  DialogContent,
+  DialogContentText,
+  DialogTitle,
+  IconButton,
+  Tooltip,
+} from '@material-ui/core';
+import _ from 'lodash';
+import { useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import { useDispatch } from 'react-redux';
+import { apply } from '../../../lib/k8s/apiProxy';
+import { KubeObject } from '../../../lib/k8s/cluster';
+import { clusterAction } from '../../../redux/actions/actions';
+import AuthVisible from './AuthVisible';
+
+interface RestartButtonProps {
+  item: KubeObject;
+}
+
+export function RestartButton(props: RestartButtonProps) {
+  const { item } = props;
+  const { t } = useTranslation();
```

Comment:
This needs a namespace. 'frequent' I guess.

Reply:
updated